tsconfig.json

```diff
@@ -6,6 +6,15 @@
     "module": "ESNext",
     "skipLibCheck": true,
 
+    "baseUrl": "./src",
+    "paths": {
+      "components/*": ["components/*"],
+      "helpers/*": ["helpers/*"],
+      "hooks/*": ["hooks/*"],
+      "stories/*": ["stories/*"],
+      "types": ["types.ts"]
+    },
```

Comment:
This change tells TypeScript the rules and allows "component/ComponentName" imports, etc.